tests/testthat/dummy_packages/missing_dep/R/foo.R

```diff
@@ -0,0 +1,7 @@
+downstream <- function() {
+  a_really_really_long_local_object_name <- 1
+  aSilLLyObjEctNaME <- 2
+
+  myFancyUpstream::upstreams_really_long_imported_object_name()
+  myFancyUpstream::uPStreams_SilLY.objecT.Name()
```

Comment:
Also add a case of `importFrom()`?

Reply:
oh, good call. turns out we need to skip more... need to investigate further:

https://github.com/r-lib/lintr/blob/cc9a221763ffe01492eb3b4672915a78ea8c67f9/R/namespace.R#L29